frontend/vite.config.js

```diff
@@ -32,6 +32,14 @@ const config = {
 	build: {
 		sourcemap: true
 	}
+
+	/* Trying
+
+	ssr: {
+		noExternal: ['@ckeditor/*'],
+	}
+
+ end trying*/
```

Comment:
We'll have to check this as this resolved an error only present in preview or only even in final build